packages/phrases/src/locales/en/translation/admin-console/sign-in-exp.ts

```diff
@@ -38,6 +38,22 @@ const sign_in_exp = {
     slogan: 'Slogan',
     slogan_placeholder: 'Unleash your creativity',
   },
+  sign_up_and_sign_in: {
+    identifiers: 'Sign up identifiers', // UNTRANSLATED
```

Comment:
Is the "UNTRANSLATED" tag indicates that this is not reviewd by @BTX26?

Reply:
Yes, It is also used to mark the newly added but not determined translations.

Reply:
Oh, this field in `en` is determined; Tag removed.